validator.js

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2019-2020 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
```

Comment:
no need for use strict in es6 module, I think this might apply to some other client libraries as well.